index.html

```diff
@@ -0,0 +1,332 @@
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+
+
+                <div class="projectBox">
+                    <div class="boxContainer">
+                        <div class="picture">
+                            <img class="picturesPreview" src="/img/pic5.png" alt="picture1">
+                        </div>
+                        <div class="textBox">
+                            <div class="boxTitle">
+                                Project name goes here
+                            </div>
+                            <div class="codingLangButtons">
+                                <div class="codingLangBackground">
+                                    HTML/CSS
+                                </div>
+                                <div class="codingLangBackground">
+                                    Ruby on Rails
+                                </div>
+                                <div class="codingLangBackground">
+                                    JavaScript
+                                </div>
+                            </div>
+                            <div class="seeThisProject">
+                                <img class="stpButton" src="/img/stpButton.png" alt="geometryDivisor">
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+
+
+
+                <div class="projectBox">
+                    <div class="boxContainer">
+                        <div class="picture">
+                            <img class="picturesPreview" src="/img/pic6.png" alt="picture1">
+                        </div>
+                        <div class="textBox">
+                            <div class="boxTitle">
+                                Project name goes here
+                            </div>
+                            <div class="codingLangButtons">
+                                <div class="codingLangBackground">
+                                    HTML/CSS
+                                </div>
+                                <div class="codingLangBackground">
+                                    Ruby on Rails
+                                </div>
+                                <div class="codingLangBackground">
+                                    JavaScript
+                                </div>
+                            </div>
+                            <div class="seeThisProject">
+                                <img class="stpButton" src="/img/stpButton.png" alt="geometryDivisor">
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+            </section>
+
+            </section>
+
+            <section id="projects">
+
+            </section>
+
+            <about id="about">
+                About
+            </about>
+
```

Comment:
- [ ] kindly consider using existing tags in HTML this will make project more professional